test/internals/toString.test.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const Fastify = require('../..')
+const fp = require('fastify-plugin')
+
+test('toString method should exist', t => {
+  t.plan(1)
+  const fastify = Fastify()
+  t.ok(fastify.toString)
+})
+
+test('should return valid string', t => {
+  t.plan(1)
+  const fastify = Fastify()
+  fastify.get('/', (request, reply) => reply.send({ text: '/' }))
+  fastify.post('/test', (request, reply) => reply.send({ text: 'test' }))
+
+  fastify.use(function helloWorld (instance, req, res) {
+    console.log('Hello, World!')
+  })
+
+  fastify.use('/abc', function routeABC (instance, req, res) {
+    console.log('ABC')
+  })
+
+  fastify.register(fp((instance, opts, next) => {
+    instance.decorate('anonymousPlugin', (a, b) => a + b)
+    next()
+  }))
+
+  fastify.register(fp(function standardPlugin (instance, opts, next) {
+    instance.decorate('standardPlugin', (a, b) => a - b)
+    next()
+  }))
+
+  const STR = `Fastify Instance toString
```

Comment:
I do not think we should start with `Fastify Instance toString`. Maybe should we just avoid a title?